### PR TITLE
Add Meson Protocol

### PR DESCRIPTION
**Pull request by wyf-ACCEPT**
Add Meson, a cross-chain swap protocol, into Defillama dashboard

### CODE REVIEW

File: src/adapters/meson/index.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  eventParams.push(depositParams, withdrawalParams)
+  return async (fromBlock: number, toBlock: number) =>
+    getTxDataFromEVMEventLogs("meson", chain as Chain, fromBlock, toBlock, eventParams)
+}
+
+const adapter: BridgeAdapter = {
+  ethereum: constructParams("ethereum"),
+  polygon: constructParams("polygon"),
+  fantom: constructParams("fantom"),
+  avax: constructParams("avax"),
```
Comment (vrtnd) [inline src/adapters/meson/index.ts:25]:
```suggestion
  avalanche: constructParams("avax"),
```

File: src/data/bridgeNetworkData.ts [hunk 2]
```diff
@@ -417,4 +417,25 @@ export default [
   //     avalanche: "avax", // this is needed temporarily, need to fix and remove
   //   },
   // },
+  {
+    id: 28,
+    displayName: "Meson",
+    bridgeDbName: "meson",
+    iconLink: "",
```
Comment (vrtnd) [inline src/data/bridgeNetworkData.ts:424]: Need icon

**Comment by vrtnd** [comment]
looks gud just need to add icon
thanks for pr

**Comment by wyf-ACCEPT** [comment]
> looks gud just need to add icon thanks for pr

Hi I've just upload the Meson icon and commited the PR here: https://github.com/DefiLlama/icons/pull/597

**Pull request merged**